test/e2e/run_test.go

```diff
@@ -803,6 +803,17 @@ USER mail`
 		Expect(isSharedOnly).Should(BeTrue())
 	})
 
+	It("podman run --procfs-opts", func() {
+		session := podmanTest.Podman([]string{"run", "--procfs-opts", "nosuid,exec,gid=10", fedoraMinimal, "findmnt", "-noOPTIONS", "/proc"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		output := session.OutputToString()
+		// gid= is showed with the user namespace mappings, so ignore the value
+		Expect(output).To(ContainSubstring("gid="))
+		Expect(output).To(ContainSubstring("nosuid"))
+		Expect(output).To(Not(ContainSubstring("exec")))
```

Comment:
looks odd that exec is not expected but the other values are?

Reply:
the kernel won't show the default `exec`.  It would show `noexec` if it was specified